docs/architecture/frontend_architecture.rst

```diff
@@ -27,43 +27,43 @@ For example:
 .. code-block:: none
 
   kolibri/
-    core/                       # core (shared) items
+    core/                             # core (shared) items
       assets/
         src/
-          core-base.vue         # global base template, used by apps
-          core-modal.vue        # example of another shared component
-          core-global.scss      # globally defined styles, indluded in head
-          core-theme.scss       # style variable values
-          font-noto-sans.css     # embedded font
+          CoreBase.vue                # global base template, used by plugins
+          CoreModal.vue               # example of another shared component
+          core-global.scss            # globally defined styles, included in head
+          core-theme.scss             # style variable values
+          font-noto-sans.css          # embedded font
         test/
-          ...                   # tests for core assets
+          ...                         # tests for core assets
     plugins/
-      learn                     # learn plugin
+      learn                           # learn plugin
         assets/
           src/
-            vue/
-              index.vue         # root view
-              some-page.vue     # top-level client-side page
-              another-page/     # top-level client-side page
+            views/
+              LearnIndex.vue          # root view
+              SomePage.vue            # top-level client-side page
+              AnotherPage/            # top-level client-side page
                 index.vue
-                child.vue       # child component used only by parent
-              shared.vue        # shared across this plugin
-            app.js              # instantiate learn app on client-side
+                Child.vue             # child component used only by parent
+              Shared.vue              # shared across this plugin
+            app.js                    # instantiate learn app on client-side
             router.js
             store.js
           test/
             app.js
       management/
         assets/
           src/
-            vue/user-page.vue   # nested-view
-            vue/index.vue       # root view
-            app.js              # instantiate mgmt app on client-side
+            views/UserPage.vue        # nested-view
+            views/ManagementIndex.vue # root view
+            app.js                    # instantiate mgmt app on client-side
```

Comment:
I think the most significant change since this was first written is how we deal with vuex - would you mind adding that?